.idea/misc.xml

```diff
@@ -6,6 +6,9 @@
     </list>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="JavaScriptSettings">
```

Comment:
What's up with this?

Reply:
IntellIJ is weird, strange side effect of having your .idea folder on VCS (Version Control)